portal-impl/src/com/liferay/portal/tools/deploy/ExtDeployer.java

```
@@ -43,7 +44,10 @@ else if (arg.endsWith(".jar")) {
 			}
 		}
 
-		StreamUtil.cleanUp(new ExtDeployer(wars, jars));
+		try (ExtDeployer temp = new ExtDeployer(wars, jars)) {
+		}
+		catch (IOException ioe) {
```

Why does this one have an empty catch when the one in BaseDeployer.java logs?